app/app.rb

```diff
@@ -78,3 +78,29 @@
   status 400
   body {}
 end
+
+post '/prayers' do
+  email = request.fetch_header('P4L-email')
+  user = User.find_by_email(email)
+
```

Comment:
Should this be changed to match what the header looks like coming from React Native?

When I send the header ("P4L-email") from the front end it changes the key to "HTTP_P4L_EMAIL".
Or is this maybe a problem with how I'm sending the headers?

Reply:
You're absolutely right. I'm getting ready to re-work the whole auth thing so I just didn't want to mess with it yet.
Why don't I fix this on the next PR where I add the auth middleware?

Reply:
Sounds good to me!! Just thought I would mention it